lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -611,6 +612,7 @@ void SPIRVToOCLBase::visitCallBuildNDRangeBuiltIn(CallInst *CI, Op OC,
 
 void SPIRVToOCLBase::visitCallGenericCastToPtrExplicitBuiltIn(CallInst *CI,
                                                               Op OC) {
+  assert(CI->getCalledFunction() && "Unexpected indirect call");
```

Comment:
Yeah, I think my previous comment works for all the assertions in this patch. They don't seem to be necessary.

Reply:
Indeed, but please check the PR description :)
There are several visit ... functions in the file, some of them have the assertions, others don't. So I'd say we'd better to either remove all of them or add new assertions. I'm kind of on adding extra, to express 'not-trust-function inputs' strategy. But yeah, all of these assertions aren't needed actually.